apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/Disclaimer/useDisclaimerViewModel.ts

```diff
@@ -51,8 +52,12 @@ export default function useDisclaimerViewModel(goToApp: (manifestId: string) =>
   );
 
   const toggleCheck = useCallback(() => {
+    track("web3hub_disclaimer_toggle", {
+      component: `Disclaimer`,
+      action: `turn checkbox to ${isChecked ? "off" : "on"}`,
+    });
```

Comment:
Not sure we actually need to know this, even more as we don't even look if we succeed going to the app
Using the `TrackScreen` should be enough for us to know if the user is able to navigate to different screens